app/controllers/collections_controller.rb

```diff
@@ -4,12 +4,17 @@
 
 class CollectionsController < ApplicationController
   before_action :load_and_authorize_collection, except: %i[index new create]
+  before_action :set_option, only: %i[index]
 
   def index
-    @collections = Collection.member(current_user).or(Collection.public_access).includes(:users, tasks: %i[user groups])
-      .order(id: :asc)
-      .paginate(page: params[:page], per_page: per_page_param)
-      .load
+    @collections = case @option
+                     when 'favorites'
```

Comment:
By the way, this could also be constants, e.g. `Collection::OPTION::FAVORITES`, like you did previously (don't remember where I saw this).

Reply:
I see but you mean, but don't remember using it anywhere and couldn't find it - I would like to keep it as it is for now